samples/nextjs/src/Layout.tsx

```diff
@@ -59,7 +59,7 @@ const Layout = ({ context }: LayoutProps): JSX.Element => {
   // Note the context object type here matches the initial value in [[...path]].tsx.
   useEffect(() => {
     updateSitecoreContext && updateSitecoreContext(context);
-  }, [context]);
+  }, [updateSitecoreContext, context]);
```

Comment:
What is the reason to add `updateSitecoreContext` here?

Reply:
This was to satisfy the ['exhaustive-deps' lint rule](https://github.com/facebook/react/issues/14920). e.g. "React Hook useEffect has a missing dependency: 'updateSitecoreContext'. Either include it or remove the dependency array."

`react-hooks` ESLint plugin was brought in as part of [`eslint-config-next`](https://github.com/vercel/next.js/blob/canary/packages/eslint-config-next/index.js).